### DISCUSSION TITLE
Icon request: fa-kick-scooter (Foot-propelled skateboard with a handlebar)

### DISCUSSION THREAD

**Discussion by martindubenet** [FortAwesome/Font-Awesome#21180] [Icon Requests]
With all the news coverage regarding the new electric scooters (« trottinette » as we call them in French) from LIME, BIRD and others.

![fa-scooter_scketch](https://user-images.githubusercontent.com/4236385/63881320-9518f500-c99d-11e9-846a-f61d4ee6d4af.jpg)

Here I also proposed the #15448 `fa-motor-scooter` to eliminate the confusion between those two transport vehicules that use the same terms in English.

**Comment by go-jan** [comment]
yes please +1
how can we vote this up @tagliala ?

**Comment by tagliala** [comment]
> how can we vote this up @tagliala ?

Add a 👍🏼 reaction to the first post. You can also share this request on a social or on a forum for a community that could be interested in having this icon implemented, but please remember that commenting with +1 is against our contributing guidelines, so be specific that the upvotes need to be done through reactions

You may also be interested in taking a look at our [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🛴 [U+1F6F4](https://emojipedia.org/emoji/%F0%9F%9B%B4/) Kick Scooter

**Comment by akwasin** [comment]
Whats the latest on this? I didnt find anything on the leaderboard related to e-scooter, kickscooter or kickbike

**Comment by divadsn** [comment]
+1, how is it possible that in 2024 there is still no kickscooter icon despite their popularity...

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by jckhwkns** [comment]
Amazing that it takes this long to deliver on these requests...

**Comment by jckhwkns** [comment]
![fa-scooter](https://github.com/user-attachments/assets/89e9754a-78dd-49e7-b423-b668a9f032c8)

**Comment by sensibleworld** [comment]
Hey folks, we missed updating this issue to say that this icon was created already: https://fontawesome.com/icons/scooter?f=classic&s=solid
